docker/zap_common.py

```diff
@@ -244,7 +244,7 @@ def cp_to_docker(cid, file, dir):
 
 
 def running_in_docker():
-    return os.path.exists('/.dockerenv') or os.path.exists('/run/.containerenv')
+    return os.path.exists('/.dockerenv') or os.path.exists('/run/.containerenv') or os.environ.get("IS_CONTAINERIZED") is not None
```

Comment:
Should this actually check the value? Can anyone foresee a scenario where someone might purposefully set it to false or some value other than true?

Reply:
Good point. I've added an actual check on the value. Just to be sure that its always used consistently.

At the moment there is no real use to setting it to `false` as it would be overwritten by the path checks. Might be possible to extend this in the future when needed.